app/assets/javascripts/school_administrator_dashboard/tardies_dashboard/SchoolTardiesDashboard.js

```diff
@@ -172,7 +172,9 @@ export default class SchoolTardiesDashboard extends React.Component {
       return homeroomTardyEvents[b] - homeroomTardyEvents[a];
     });
     const homeroomSeries = homerooms.map((homeroom) => {
-      return homeroomTardyEvents[homeroom];
+      const color = (homeroom === this.state.selectedHomeroom)? 'orange' : null;
+      const y = homeroomTardyEvents[homeroom];
+      return {homeroom, y, color};
```

Comment:
I figure that `y` and `color` are options to Highcharts but wonder if the `homeroom` value is doing anything here, since there's no other change in this PR to read that value.

Reply:
I guess this isn't really necessary. As I read the [highcharts docs](https://www.highcharts.com/docs/chart-concepts/series), I thought the series name was needed to get the colors to work, but it seems to work just as well without them.